include/wx/generic/gridsel.h

```diff
@@ -151,6 +162,16 @@ class WXDLLIMPEXP_CORE wxGridSelection
     wxGrid                              *m_grid;
     wxGrid::wxGridSelectionModes        m_selectionMode;
 
+    // Used by wxGrid::DrawSelectionOverlay() to draw a:
+    //
+    // - Simple rectangle (using wxDC::DrawRectangle() if it is empty and the bounding box is valid.
+    // - Simple polygon (using wxDC::DrawPolygon()) if it represents a simple polygon.
+    // - Poly-polygon (using wxDC::DrawPolyPolygon()) if it consists of multi-polygons.
+    //
+    std::unique_ptr<wxGridPrivate::PolyPolygon> m_polyPolygon;
```

Comment:
We should `#include <memory>` in this file instead of relying on it being done implicitly in some other header.

Reply:
Done